app/models/publisher.rb

```diff
@@ -210,6 +210,10 @@ def sync_wallet_connections
     end
   end
 
+  def uphold_publisher?
```

Comment:
```suggestion
  def is_selected_wallet_provider_uphold?
```

Prefer renaming to something more understandable at-a-glance.

Reply:
Fixed.